src/peer/room.js

```diff
@@ -87,6 +91,47 @@ class Room extends EventEmitter {
     return true;
   }
 
+  /**
+   * Send a message to the room with a delay so that the transmission interval does not exceed the limit.
+   * @param {object} msg - The data to send to this room.
+   * @param {string} key - The key of broadcast event.
+   */
+  _sendMessage(msg, key) {
```

Comment:
It seems that if you use `_sendMessage`, you can only send sentences, so I thought `_sendData` would be easier to understand.